<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInterActiveCtrlWdg</class>
 <widget class="QWidget" name="CInterActiveCtrlWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CInterActiveCtrlWdg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget{
	background-color:rgb(28,28,28);
}
QLabel#label{
	color:rgb(127,127,127);	
}

QPushButton#pushButton_cameraList{
	border:0px;	
	background-color:rgb(28,28,28);
}
QPushButton#pushButton_cameraList:hover{
	background-color:rgb(50,50,50);
}
QPushButton#pushButton_cameraList:pressed{
	border:0px;	
	background-color: rgb(40, 40, 40);
}

QPushButton#pushButton_micList{
	border:0px;	
	background-color:rgb(28,28,28);
}
QPushButton#pushButton_micList:hover{
	background-color:rgb(50,50,50);
}
QPushButton#pushButton_micList:pressed{
	border:0px;	
	background-color: rgb(40, 40, 40);
}

QPushButton#pushButton_playerList{
	border:0px;	
	background-color:rgb(28,28,28);
}
QPushButton#pushButton_playerList:hover{
	background-color:rgb(50,50,50);
}
QPushButton#pushButton_playerList:pressed{
	border:0px;	
	background-color: rgb(40, 40, 40);
}

QPushButton#pushButton_LeaveActive{
	border-image: url(:/interactivity/leaveLive);
}
QPushButton#pushButton_LeaveActive:hover{
	border-image: url(:/interactivity/leaveLiveHover);
}
QPushButton#pushButton_LeaveActive:disabled{
	border-image: url(:/interactivity/leave_live_unenable.png);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_cameraList" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ToolButton" name="pushButton_playFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>40</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labMember">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ToolButton" name="btnRecord">
           <property name="minimumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>record</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ToolButton" name="pushButton_startLive">
           <property name="text">
            <string>start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toSpeak">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border-image: url(:/toolWidget/img/toolWidget/toSpeak.png) 0 182 0 0;
	color: rgb(255, 249, 249);
}
QPushButton:hover{
	border-image: url(:/toolWidget/img/toolWidget/toSpeak.png) 0 91 0 91;
	color: rgb(255, 253, 253);
}

QPushButton:pressed{
	border-image: url(:/toolWidget/img/toolWidget/toSpeak.png) 0 182 0 0;
	color: rgb(255, 253, 253);
}

QPushButton:disabled{		
   border-image: url(:/toolWidget/img/toolWidget/toSpeak.png)0 182 0 0;
	color: rgb(255, 216, 216);
}

</string>
           </property>
           <property name="text">
            <string>申请上麦</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ToolButton" name="pushButton_Camera">
           <property name="minimumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="text">
            <string>camera</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ToolButton" name="pushButton_cameraList">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hztLiveTool">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ToolButton" name="pushButton_mic">
           <property name="minimumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="text">
            <string>mic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ToolButton" name="pushButton_micList">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_micList" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="spacerPlayer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ToolButton" name="pushButton_player">
           <property name="minimumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>44</width>
             <height>37</height>
            </size>
           </property>
           <property name="text">
            <string>player</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ToolButton" name="pushButton_playerList">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_playerList" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolButton</class>
   <extends>QPushButton</extends>
   <header location="global">ToolButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
